pages/common/csvlook.md

```diff
@@ -0,0 +1,16 @@
+# csvlook
+
+> Render a CSV file in the console as a fixed-width table.
+> Included in csvkit.
+
+- View a CSV file:
+
+`csvlook {{data.csv}}`
+
+- View a CSV file with _less_ for easy scrolling:
```

Comment:
True. csvlook really doesn't do anything besides the most basic first example. Should I cut it down to just that (or just remove it entirely even)?